lib/tasks/data_migrations/migrate_contact_metadata_to_native_fields.rb

```diff
@@ -0,0 +1,42 @@
+namespace :data_migrations do
+  task migrate_contact_metadata_to_native_fields: :environment do
+    Account.find_each do |account|
+      puts "Migrating contacts for account: #{account.id}"
+
+      account.contacts.find_each do |contact|
+        ApplicationRecord.transaction do
+          contact.iso_country_code = PhonyRails.country_from_number(contact.msisdn)
+          contact.language_code = contact.metadata["language_code"]
+          contact.save!
+
+          # EWS Cambodia
+          contact.metadata.fetch("commune_ids", []).each do | commune_id |
+            commune = Pumi::Commune.find_by_id(phone_call_metadata(:commune_code))
```

Comment:
This line might return nil. I think there will be some invalid communes in there

Reply:
Interesting, it's was not assigned by the user, but still there are some invalid codes? We will add a check to skip it.